python/GangaDirac/Dockerfile

```diff
@@ -1,34 +1,28 @@
-FROM alexanderrichards/dirac_ui:latest
+# FROM taking build arg requires at least
+#   Docker version 17.05.0-ce, build 89658be
+ARG ganga_version=latest
+FROM ganga-core:$ganga_version
```

Comment:
When this docker container is built automatically, is there a danger that you get out of sync? So the 6.5.3 tag is created. Could the notification to docker to build a new container now  happen such that the Dirac one is done first (and thus picking the 6.5.2 tag as being the "latest".